FAQ.md

```diff
@@ -381,3 +380,10 @@ it can [cause
 problems](https://groups.google.com/d/msg/golang-nuts/AnMr9NL6dtc/UnyUUKcMCAAJ).
 If your dependers are using `dep`, this is not a concern, as `dep` takes care of
 stripping out nested `vendor` directories.
+
+## How do I configure a dependency that doesn't tag its releases?
+
+Add a constraint to your manifest that specifies `branch: "master"`
+for the dependency. `dep ensure` will deduce the current revision of your
```

Comment:
eh, one last tiny nit - we often use "deduce" to mean a specific thing in the dep/gps world. there's nothing _wrong_ with this usage here, but if you did s/determine/deduce/, it would help avoid any possible inferred relationship forming in the mind of the reader between the deduction subsystem (if they know about it) and figuring out the underlying rev of a branch.